.github/workflows/release.yml

```diff
@@ -28,12 +34,13 @@ jobs:
 
       - name: Build, test and push
         run: |
-          echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
-          docker build -t ppiper/node-browsers:${{ env.PIPER_version }} .
-          docker build -t ppiper/node-browsers:${{ env.PIPER_version }}-node12 node12
-          docker push ppiper/node-browsers:${{ env.PIPER_version }}
-          docker push ppiper/node-browsers:${{ env.PIPER_version }}-node12
-
+          echo ${{ secrets.DH_PW }} | docker login -u ${{ secrets.DH_USER }} --password-stdin
```

Comment:
Did the secret name change happened occasionally?

Reply:
this was my mistake, I've mixed the writings up and added both ways to the repo, so it does no harm. I've changed back to the long form which I like better.